### PR TITLE
Clean up mobilityAuto

### PR DESCRIPTION
**Pull request by elleryjh**

### CODE REVIEW

File: src/main/java/frc/robot/commands/auto/Autos.java [hunk 1]
```diff
@@ -23,6 +24,8 @@ public class Autos {
   private final Arm arm;
   private final Orchestrator orchestrator;
 
+  private double MOBILITY_DRIVE_DISTANCE = 6.75;
```
Comment (admlvntv) [inline src/main/java/frc/robot/commands/auto/Autos.java:27]: Put this in meters (using Units class)

File: src/main/java/frc/robot/commands/auto/Autos.java [hunk 2]
```diff
@@ -119,44 +122,26 @@ private Translation2d getNotePositions(int index, boolean centerNotes) {
...
+                        .withTimeout(1)
+                        .andThen(driveXDistance(MOBILITY_DRIVE_DISTANCE).withTimeout(5))),
+                () -> position));
+  }
+
+  private Command driveXDistance(double distance) {
+    return Commands.defer(
+        () ->
+            new StraightDriveToPose(
+                Units.feetToMeters(AllianceFlipUtil.applyRelative(distance)), 0, 0, drive),
```
Comment (admlvntv) [inline src/main/java/frc/robot/commands/auto/Autos.java:143]: accept only meters

**Review by admlvntv — CHANGES_REQUESTED** [review]
unit fixes

**Review by Aradig26 — APPROVED** [review]
LGTM

**Pull request merged**